src/main/resources/messages_en.properties

```diff
@@ -409,19 +409,19 @@ lot.input.list.stock.ids.null.or.empty=StockIds are mandatory, please check for
 lot.input.list.stock.ids.duplicated=We have found some duplicated Stock Ids, please fix them and try again
 lot.input.list.stock.ids.invalid=We have found the following StockId(s): {0} in the system. Please correct them and try again.
 lot.input.list.notes.length=The max length of Notes is 255 characters
-
 selected.lots.with.no.unit={0} selected lots do not have units defined to perform a withdrawal. Please review.
 selected.lots.closed={0} selected lots are closed. Please review 
 lot.amount.invalid=Invalid amount. Please review
 lot.withdrawal.input.null=Request Object can not be null
 lot.input.instructions.missing.for.units=Missing instructions for the following unit(s): {0}
 no.lots.selected=List of lots to perform an action can not be empty. Make sure that your searchId returns some lots to work with
+no.transactions.selected=List of transactions to perform an action can not be empty. Make sure that your searchId returns some transactions to work with
 lot.selection.invalid=Either provide searchRequestId or list of lots to operate with
-lots.does.not.exist=Some of the lot ids were not found. Please review 
+lots.does.not.exist=Some of the lot ids were not found. Please review
+transactions.does.not.exist=Some of the transaction ids were not found. Please review
 lots.transactions.status.confirmed.cannot.change.unit=The scale update is not possible on lots with confirmed transactions
 transaction.selection.invalid=Either provide searchId or list of transactions to operate with
-transaction.does.not.exist=Some of the transaction ids were not found. Please review 
-
+transaction.does.not.exist=Some of the transaction ids were not found. Please review
```

Comment:
@cuenyad merge this property with transaction.update.not.found.transactions. Use same message and rename the old property to be more generic